content/blog/open-source-monitoring-at-the-hpe-customer-technology-center-böblingen.md

```diff
@@ -0,0 +1,412 @@
+---
+title: Open Source Monitoring at the HPE Customer Technology Center Böblingen
+date: 2024-01-02T15:34:10.449Z
+author: "Thomas Beha "
+authorimage: /img/tb10092023.jpg
+thumbnailimage: /img/ctcracktemppower.small.png
+disable: false
+tags:
```

Comment:
Please add one additional tag: ilo-restful-api
This will allow to tie your blog post to the iLO RESTful API product page on the HPE Dev Community portal:
https://developer.hpe.com/platform/ilo-restful-api/home/